products/llbuildSwift/CMakeLists.txt

```diff
@@ -24,11 +24,32 @@ else()
   if(LIBDISPATCH_SOURCE_DIR)
     list(APPEND additional_args -I${LIBDISPATCH_SOURCE_DIR})
   endif()
+  if(CMAKE_SYSTEM_NAME STREQUAL Linux)
+    list(APPEND additional_args -Xcc;-DDEPLOYMENT_TARGET_LINUX)
+  elseif(CMAKE_SYSTEM_NAME STREQUAL Darwin)
+    list(APPEND additional_args -Xcc;-DDEPLOYMENT_TARGET_MACOSX)
+  elseif(CMAKE_SYSTEM_NAME STREQUAL Windows)
+    list(APPEND additional_args -Xcc;-DDEPLOYMENT_TARGET_WINDOWS;-Xcc;-D_DLL;-Xlinker;/NODEFAULTLIB:libcmt)
+  else()
+    message(SEND_ERROR "unknown deployment target: ${CMAKE_SYSTEM_NAME}")
+  endif()
+  if(LLBUILD_PATH_TO_SQLITE_SOURCE)
+    list(APPEND additional_args -I${LLBUILD_PATH_TO_SQLITE_SOURCE})
+  endif()
+  if(LLBUILD_PATH_TO_SQLITE_BUILD)
+    list(APPEND additional_args -L${LLBUILD_PATH_TO_SQLITE_BUILD})
+  endif()
```

Comment:
This is (partially) from @compnerd's patch that got reverted (https://github.com/apple/swift-llbuild/pull/453/files). DEPLOYMENT_TARGET has to be set to build with foundation on Windows.

Reply:
There is a pending PR #477 which should repair this.